lib/perl/centreon/script/centcore.pm

```diff
@@ -685,12 +689,16 @@ sub sendConfigFile($){
                     'on poller "' . $server_info->{name} . '" (' . $server_info->{id} . ')'
                 );
                 $cmd = "cp $origin $cfg_dir 2>&1";
-                ($lerror, $stdout) = centreon::common::misc::backtick(
+                ($lerror, $stdout, $return_code) = centreon::common::misc::backtick(
                     command => $cmd,
                     logger => $self->{logger},
                     timeout => 60
                 );
             }
+                            
+            if ($lerror != 0 || (defined($return_code)  && $return_code != 0)) {
```

Comment:
I think there's a space to remove between : `$return_code)` and `&& $return_`

Reply:
For me it's ok. We should merge that kind of stuff in one PR clean code.